tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -20,6 +20,12 @@ DEEPBUFFER_FW_DMA_MS=100"
 "cavs-sdw\;sof-mtl-rt711-4ch\;PLATFORM=mtl,NUM_DMICS=4,DMIC0_ID=2,DMIC1_ID=3,NUM_HDMIS=0,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt711-4ch.bin,DEEPBUFFER_FW_DMA_MS=100"
 
+# SDW + DMIC + HDMI with SDW_AMP_FEEDBACK
+"cavs-sdw\;sof-mtl-cs42l42-l0-max98363-l2-4ch\;PLATFORM=mtl,\
+NUM_SDW_AMPS=2,SDW_SPK_STREAM=SDW2-Playback,SDW_SPK_IN_STREAM=SDW2-Capture,\
+NUM_DMICS=4,DMIC0_ID=2,DMIC1_ID=3,NUM_HDMIS=4,DEEPBUFFER_FW_DMA_MS=100,\
```

Comment:
You need `PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1` as `NUM_DMICS=4`

Reply:
Ack, will update in next version.